apps/mull-ui-e2e/src/integration/US-1.1.spec.ts

```diff
@@ -38,8 +38,33 @@ frameSizes.forEach((frame) => {
     it('should type into the event description input', () => {
       cy.get('#description').type('test description').should('have.value', 'test description');
     });
-    it('should type into the event location input', () => {
-      cy.get('#location').type('test location').should('have.value', 'test location');
+
+    it('should type into the event location modal and return autocompleted address', () => {
+      cy.get('#location').click();
+      cy.get('#location-input-field').should('be.visible');
+      const res = '845 Rue Sherbrooke';
+      cy.get('#location-input-field').type('845 Rue Sherbrooke');
+      cy.get('#location-input-field-option-0', { timeout: 3500 }).should(
+        'have.text',
+        '845 Rue Sherbrooke, Montreal, QC, Canada'
+      );
+      cy.get('#location-input-field-option-0').click();
+    });
+
+    it('should click current location on event location modal', () => {
+      cy.get('#location').click();
+      cy.get('#location-input-field').should('be.visible');
+      cy.get('#location-input-field-option-0').click();
+      cy.get('#location')
+        .invoke('val')
+        .should('contain', 'longitude')
+        .should('contain', 'latitude');
+    });
```

Comment:
Test needs to be updated since we don't show `longitude` and `latitude` anymore

Reply:
Addressed